website/README.md

```diff
@@ -8,6 +8,60 @@ This website was created with [Docusaurus](https://docusaurus.io/).
 - [Adding Content](#adding-content)
 - [Full Documentation](#full-documentation)
 
+# Contribute
+If you wish to contribute to this project you can do so by Forking this repository and creating a new branch for your changes based on the latest development branch.
+
+```
+  git clone your-forked-repo-url
+  cd repo_name  
+  git checkout develop
+  git checkout -b "prefix/your-cool-new-feature"
+```
+or if you are using [gitflow](https://github.com/petervanderdoes/gitflow-avh)
```

Comment:
using git flow for forked projects doesnt make sense. Only when working on the "main" repo. Rather make it a section for non-fork workflow. (e.g. for internal use)